src/pomodoro/containers/PomodoroControls.js

```diff
@@ -74,40 +74,68 @@ class PomodoroControl extends React.PureComponent {
   }
 }
 
-const PomodoroControls = ({ status, play, pause, stop, skip }) => (
-  <Container>
-    <PomodoroControl
-      icon='stop'
-      size='small'
-      design='tertiary'
-      onClick={stop}
-      disabled={Boolean(status === 'IDLE')}
-    />
-
-    {status === 'RUNNING' ? (
-      <PomodoroControl
-        icon='pause'
-        onClick={pause}
-        width='28'
-        height='28'
-      />
-    ) : (
-      <PomodoroControl
-        icon='play'
-        onClick={play}
-        width='38'
-        height='38'
-      />
-    )}
-
-    <PomodoroControl
-      icon='skip'
-      size='small'
-      design='tertiary'
-      onClick={skip}
-    />
-  </Container>
-)
+class PomodoroControls extends React.PureComponent {
+
+  constructor (props) {
+    super(props)
+    this.handleKeyboard = this.handleKeyboard.bind(this)
```

Comment:
As we are using class field property to define the `handleKeyboard` method as an arrow function, this bind is not needed. 

In arrow functions, the `this` keyword has lexical scope, which means it always refers to the current scope, not to the scope where it is being called, that's why we don't need the `bind` call.

Let me know if you have doubts about it and I can explain better to you.